.github/workflows/test-semantic.yml

```diff
@@ -11,6 +11,6 @@ jobs:
   semantic_pr:
     runs-on: ubuntu-latest
     steps:
-      - uses: amannn/action-semantic-pull-request@v1.2.0
```

Comment:
a big bump here. what the new action bring in? any breaking changes?

Reply:
only some fixes and new config parameters. But don't have any break changes